cmd/ec2.go

```diff
@@ -6,37 +6,75 @@ package cmd
 import (
 	"fmt"
 	"os"
+	"strings"
 	"text/tabwriter"
+
 	"github.com/spf13/cobra"
-	"github.com/surajincloud/awsctl/pkg"
+	"github.com/surajincloud/awsctl/pkg/ec2"
 )
 
-// ec2Cmd represents the ec2 command
 var ec2Cmd = &cobra.Command{
 	Use:   "ec2",
 	Short: "Print ec2 related information",
 	Long: `For example,
 		awsctl get ec2`,
-	Run: getEC2command,
+	Run: func(cmd *cobra.Command, args []string) {
+		tag, _ := cmd.Flags().GetString("tags")
+		if tag != "" {
+			getEc2instanceTag(cmd, args)
```

Comment:
since both the functions are pretty much the same with minor feature change such as tag, can we use only one function and also pass `flags` to the same function.

something like getEC2(cmd, args,flags)

Reply:
ignore in fact, you can fetch flags from `cmd` itself so no need of checking tags at this stage